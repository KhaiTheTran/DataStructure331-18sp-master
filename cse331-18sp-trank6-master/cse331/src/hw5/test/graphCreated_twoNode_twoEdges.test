# Test graphCreated_twoNode_twoEdges

# Create a graph
CreateGraph graphCreated_twoNode_twoEdges

# Add two nodes
AddNode graphCreated_twoNode_twoEdges xx
AddNode graphCreated_twoNode_twoEdges yy

# Add two edges
AddEdge graphCreated_twoNode_twoEdges xx xx xxxx
AddEdge graphCreated_twoNode_twoEdges xx yy yyxx

# Print the nodes in the graph and the outgoing edges from xx,
# and outgoing from yy
ListNodes graphCreated_twoNode_twoEdges
ListChildren graphCreated_twoNode_twoEdges xx
ListChildren graphCreated_twoNode_twoEdges yy