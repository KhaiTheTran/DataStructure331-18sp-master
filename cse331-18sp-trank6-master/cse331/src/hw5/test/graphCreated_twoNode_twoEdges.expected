# Test graphCreated_twoNode_twoEdges

# Create a graph
created graph graphCreated_twoNode_twoEdges

# Add two nodes
added node xx to graphCreated_twoNode_twoEdges
added node yy to graphCreated_twoNode_twoEdges

# Add two edges
added edge xxxx from xx to xx in graphCreated_twoNode_twoEdges
added edge yyxx from xx to yy in graphCreated_twoNode_twoEdges

# Print the nodes in the graph and the outgoing edges from xx,
# and outgoing from yy
graphCreated_twoNode_twoEdges contains: xx yy
the children of xx in graphCreated_twoNode_twoEdges are: xx(xxxx) yy(yyxx)
the children of yy in graphCreated_twoNode_twoEdges are: